scripts/test.sh

```diff
@@ -0,0 +1,4 @@
+./scripts/start-substrate.sh --test-db
```

Comment:
There should be a `make test` command for running this. Other than that, we're good to go!